pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,57 @@
+# Inclusive Language
```

Comment:
@BhaswatiRoy Kindly add metadata to this page so that the scripts of our website will easily recognize it as a page.

Reply:
done @akshatnema